Description: Bind F11 to start devhelp search for the currently word in Emacs

Index: devhelp-3.0.0/misc/devhelp.el
===================================================================
--- devhelp-3.0.0.orig/misc/devhelp.el	2010-07-02 23:02:22.000000000 +0200
+++ devhelp-3.0.0/misc/devhelp.el	2011-07-12 09:53:58.649742279 +0200
@@ -34,7 +34,7 @@
 ;; Examples:
 ;;
 ;; Bind F7 to start devhelp and search for the word at the point.
-;; (global-set-key [f7] 'devhelp-word-at-point)
+(global-set-key [f11] 'devhelp-word-at-point)
 ;;
 ;; Bind F6 to enable the automatic assistant.
 ;; (global-set-key [f6] 'devhelp-toggle-automatic-assistant)
